common/content/commandline.js

```diff
@@ -19,8 +19,10 @@ const CommandLine = Module("commandline", {
 
         this._callbacks = {};
 
-        storage.newArray("history-search", { store: true, privateData: true });
-        storage.newArray("history-command", { store: true, privateData: true });
+        ["search", "command"].forEach(function (mode) {
+            storage.newArray(liberator.storeName(mode, liberator.isPrivateWindow()),
+                { store: true, privateData: true });
```

Comment:
If divite private mode, store is false when private mode.
and privateData may be not necessary.